Makefile

```diff
@@ -120,8 +128,11 @@ dev-push:
 ###################################
 
 # Create YAML file for deployment
-prod-yaml: | $(KUSTOMIZE)
-	CAPV_MANAGER_IMAGE=${PRODUCTION_IMG} cmd/clusterctl/examples/vsphere/generate-yaml.sh
+prod-yaml: clusterctl-tools
```

Comment:
Hi @andrewsykim,

This should be an order-only dependency. Please change it to:

```make
prod-yaml: | clusterctl-tools
```